### ISSUE TITLE
[Bug]: 1.18 Mixins crash when playing alongside some other mods

### ISSUE THREAD

**Issue by Raidobw2** [AstralOrdana/Mores#4]
# Problem
The game will crash on initialize if SBM Jukebox Fabric and Ordana's Mores is installed.

I wrote this on your mod's issue tracker since ExampleMixin shows up in the stacktrace and as far as I could read your mod's code, you don't use mixins. SBM Jukebox uses some for the Jukebox, but I have no idea why there's a specific interaction like that. Your mod plays nice with other mods that have mixins as well, it's for some reason that one that doesn't work with it. Let me know if you think it's the other mod's problem, I'll link them a report too. Thanks in advance!

# Steps to reproduce
1. Install a new instance on the CurseForge App for Fabric 0.12.11.
2. Install SBM Jukebox Fabric from https://www.curseforge.com/minecraft/mc-mods/sbm-jukebox-fabric/files/3541465.
3. Install Ordana's More from https://www.curseforge.com/minecraft/mc-mods/ordanas-mores/files/3551196.
4. Play the game and notice it crashes directly at the beginning.

# Versions
MC: `1.18`
Fabric: `0.12.11`
SBM Jukebox Fabric: `sbm-jukebox-1.18-rc4-1.0.1.jar`
Ordana's Mores: `mores-1.2.6.jar`

# Stacktrace
With other mods and SBM Jukebox Fabric: https://pastebin.com/QZN157B4
Just the Fabric API and those 2 mods: https://pastebin.com/cA3yPAH2
Latest.log only with Fabric API and the 2 mods: https://pastebin.com/KGw7RGXy

```
MixinApplyError: Mixin [modid.mixins.json:ExampleMixin] from phase [DEFAULT] in config [modid.mixins.json] from mod [sbm-jukebox] FAILED during PREPARE
InvalidMixinException: The specified mixin 'net.fabricmc.example.mixin.ExampleMixin' was not found
ClassNotFoundException: The specified mixin 'net.fabricmc.example.mixin.ExampleMixin' was not found
at org.spongepowered.asm.mixin.transformer.MixinInfo.loadMixinClass(MixinInfo.java:1314)
```

# Screenshots
![image](https://user-images.githubusercontent.com/17010632/145670529-6ae26d9a-ecaf-4d02-acd8-62c54793a654.png)
![image](https://user-images.githubusercontent.com/17010632/145670952-ec60c86b-f8b7-4e76-86f4-cdd5cc31a057.png)

# Config files
SBMJukeboxFabric `modid.mixins.json`
```
{
  "required": true,
  "minVersion": "0.8",
  "package": "io.github.strikerrocker.jukebox.mixin",
  "compatibilityLevel": "JAVA_17",
  "mixins": [
    "MixinJukeboxBlockEntity"
  ],
  "injectors": {
    "defaultRequire": 1
  },
  "refmap": "sbm-jukebox-1.18-rc4-refmap.json"
}
```
Ordana's Mores `modid.mixins.json`
```
{
  "required": true,
  "minVersion": "0.8",
  "package": "net.fabricmc.example.mixin",
  "compatibilityLevel": "JAVA_16",
  "mixins": [
  ],
  "client": [
    "ExampleMixin"
  ],
  "injectors": {
    "defaultRequire": 1
  }
}
```

**Comment by AstralOrdana** [comment]
Thank you for the detailed report. You are correct, Mores doens't use mixins at all and I have removed the example mixin file that is unused and might be causing this conflict. I have pushed a new version that includes that fix as well as a number of minor things. It will be labeled as version 1.2.7 on Curseforge. If the issue persists, I would bring it up with the other dev team as I have done everything I can think of on my end to fix what might be causing that.

**Comment by Raidobw2** [comment]
Thanks for the reply and the quick fix wow, that's awesome.
I've updated your mod and threw in the SBM Jukebox mod with it inside a modpack I was making for my folks. This is the ultimate test!
And I can confirm it works :)
As for other changes you pushed on 1.2.7, they are great too. I used to have a script like this one in KubeJS:
![image](https://user-images.githubusercontent.com/17010632/146120491-0ef94365-573e-4a2b-aed2-7a6bbda8eec0.png)
Which would adjust your mod's recipes since it was new, I thought it was WIP. Now it's correctly implemented, I don't need the script anymore :D 
![image](https://user-images.githubusercontent.com/17010632/146120583-009b1a23-68ad-4333-bf13-c07e10070675.png)
There's only one more thing I need to manually adjust for your mod, and it's tags. But that thing is up to the mod author's vision and intended compatibility given, really up to you. In Forge there's something called "the ore dictionary", in Fabric it's called "common", it's the same thing basically.
Some mods can implement based on common tags if they want, Tech Reborn does this in its recipes for example with copper. As you see on the image above too I'm able to process your ores in that machine with the correct tag. This provides easier integration basically.
![image](https://user-images.githubusercontent.com/17010632/146121282-a5cd6bdb-a149-4e77-9522-9a8a785e448b.png)
I'm sure you can find out how to register that later on if you feel like it since you've registered a "mores" tag. Here's my script if that helps validating you've done all ores:
![image](https://user-images.githubusercontent.com/17010632/146121445-e6b8ee1d-869a-4cf9-b07a-d05236fbcdde.png)
In short, the problem is fully resolved and working, and you added an extra with the smelting and blasting recipes :)
As a suggestion if you want you can implement common tags for the ores, this will help making your mod compatible OOTB with other mods.

You can close this ticket upon read, thanks again and have a nice holiday time in advance!

**Comment by AstralOrdana** [comment]
Thank you for this incredibly detailed reply, I will look into the "common" thing and if all goes well, push out an updated version soon. Thanks again!
